ucm2/rockchip_es8316/rockchip_es8316.conf

```diff
@@ -0,0 +1,45 @@
+Syntax 3
+
+SectionUseCase."HiFi" {
+	File "HiFi.conf"
+	Comment "Play HiFi quality Music"
+}
+
+BootSequence [
```

Comment:
Ideally, we should have this config in codecs/es8316 tree (optional). But this will require a check if it does not break the bytcht-es8316 configs. @jwrdegoede  FIY

Also, I think that the affected set of controls may be reduced to set only values which are different from the defaults set by the driver (my preference).